packages/teleterm/src/services/logger/loggerService.ts

```diff
@@ -79,9 +120,15 @@ function stringifier(message: unknown[]): string {
     .join(' ');
 }
 
-type Options = {
+type FileLoggerOptions = {
   dir: string;
   name: string;
+  /**
+   * ANSI display attribute (SGR parameter) for the logger name e.g. 104 for Bright Blue background color.
+   * Logger name is printed in the terminal, only in dev mode.
+   * If not specified, the logger name will not be printed.
+   */
+  loggerNamePrintCode?: string;
```

Comment:
Any better name is welcome 😏

Reply:
I can't think of a better name. Maybe `loggerNameANSICode` but that looks weird. Idk, thats a tough one.

Reply:
What about this:

Create a palette / "enum" of colors in loggerService.ts:

```typescript
const COLORS = {
  Magenta: 45,
  Cyan: 46,
  Yellow: 43,
}
```

and then rename the arg to `backgroundColor` and it'd be used like so:

```typescript
backgroundColor: COLORS.Magenta
```

I'm not sure about the capitalization of those color names but it ultimately doesn't matter that much. I suppose the capitalized version would be closer to TypeScript's enums.

Reply:
Good idea!

Do you have anything against enums? I like them more than objects in cases like that (I can elegantly type the field) and they are used in our codebase.
I used name `loggerNameColor` because it also determines if the logger name is printed. I think it makes more sense than `backgroundColor` in this case.